docs/validators/redundancy.md

```diff
@@ -0,0 +1,48 @@
+---
+id: redundancy
+title: Node redundancy
+---
+
+Each validator node can be backed up by one or several backup nodes in order to 
+provide security of the network and to assure that a node will unlikely get jailed.
```

Comment:
Elrond Validator Nodes can be configured to have one or more hot-standby nodes. This means additional nodes will run on different servers, in synch with the main Validator node. Their role is to stand in for the main Validator node in case it fails, to ensure high availability.

Reply:
Elrond Validator Nodes can be configured to have one or more hot-standby nodes. This means additional nodes will run on different servers, in **sync** with the main Validator node. Their role is to stand **by** for the main Validator node in case it fails, to ensure high availability. ?